README.md

```diff
@@ -105,4 +106,49 @@ E.g., `url`, `email`, etc.
 
 Doing this communicates semantics to user agents and assistive tech. For example, using `type="numeric"` might trigger the appropriate keyboard state on mobile.
 
+<a name="anchor-tag-usage"></a>
+### Anchor Tag Usage
 
+#### Anchors
+```
+<a>Text</a>
+```
+
+Anchors are great for navigation or displaying hidden information on a page. Using an anchor makes the element read as clickable to a screenreader so it is the preferred solution over a `<div>` or `<span>`.
```

Comment:
This is the weird thing about anchor elements: without the href, its behavior changes--it's almost a different element. Screen readers will NOT see it as clickable. And unless an event handler is attached, clicking will have no effect.

I'd consider calling the section "links", then listing anchor as a special case at the end of the link section, since it's used a lot less frequently.